### ISSUE TITLE
TauP introduction

### ISSUE THREAD

**Issue by core-man** [seismo-learn/software#1]
let's begin by TauP

**Comment by seisman** [comment]
We may also need a section explaining the phase naming rules in TauP, but before that, we may also need a section explaining the standard phase naming conventions in "seismology".

**Comment by core-man** [comment]
Yes, do you think it is good to be included in seismolog101 or seismology? An example:

https://migg-ntu.github.io/SeisTomo_Tutorials/seismology/seismic-wave/seismic-phases.html

**Comment by seisman** [comment]
I think we should include http://www.isc.ac.uk/standards/phases/ into "seismology", but I don't want to include all the phases, as some phases are rarely seen and used.

We also need a short section in "seismology101", but after we finish "seismology".

**Comment by core-man** [comment]
A list of Philips' codes http://www.seis.sc.edu/downloads/

**Comment by seisman** [comment]
Need to wait for the official release.

**Comment by core-man** [comment]
See https://github.com/seismo-learn/software/issues/182#issuecomment-828142469. We have four new commands.

- [ ] `velplot`
- [ ] `slowplot`
- [ ] `velmerge`
- [ ] `phase`
